src/onnx/parse_multi_head_attention.cpp

```diff
@@ -210,10 +224,60 @@ struct parse_multi_head_attention : op_parser<parse_multi_head_attention>
             MIGRAPHX_THROW("MultiHeadAttention: num_heads attribute is required");
 
         int64_t num_heads = parser.parse_value(info.attributes.at("num_heads")).at<int>();
+        uint32_t verboseLevel = 4;
+
+        CHECK_STATUS(mlssSetVerboseLevel(verboseLevel));
+
+        MLSScontext context = 0;
+        MLSSstring asic = MLSS_GFX1100;
+        MLSSstring opName = MLSS_MHA;
+
+        CHECK_STATUS(mlssCreateContext(&context, asic, opName));
+
+        //CHECK_STATUS(mlssPrintParameters(context, opName));
 
         multi_head_attention_parameters params;
         check_inputs(args, num_heads, params);
 
+        MLSSuint32 batch_size = params.batch_size;
+        MLSSuint32 head_num = num_heads;
+        MLSSuint32 q_sequence_length = params.q_sequence_length;
+        MLSSuint32 kv_sequence_length = params.kv_sequence_length;
+        MLSSuint32 head_dim = params.head_size;
+        MLSSuint32 packing = 0;
+        float scale = 1 / std::sqrt(params.head_size);
+        MLSSenum data_type = MLSS_FLOAT16;
+        MLSSuint32 kvDim = 0;
+
+        CHECK_STATUS(mlssSetParameterByEnum(&context, opName, MLSS_ATTR_MHA_BATCH, &batch_size));
+        CHECK_STATUS(mlssSetParameterByEnum(&context, opName, MLSS_ATTR_MHA_QSEQ, &q_sequence_length));
+        CHECK_STATUS(mlssSetParameterByEnum(&context, opName, MLSS_ATTR_MHA_KVSEQ, &kv_sequence_length));
+        CHECK_STATUS(mlssSetParameterByEnum(&context, opName, MLSS_ATTR_MHA_KDIM, &kvDim));
+        CHECK_STATUS(mlssSetParameterByEnum(&context, opName, MLSS_ATTR_MHA_VDIM, &kvDim));
+        CHECK_STATUS(mlssSetParameterByEnum(&context, opName, MLSS_ATTR_MHA_SIZEHEADS, &head_dim));
+        CHECK_STATUS(mlssSetParameterByEnum(&context, opName, MLSS_ATTR_MHA_PACKING, &packing));
+        CHECK_STATUS(mlssSetParameterByEnum(&context, opName, MLSS_ATTR_MHA_HEADCOUNT, &head_num));
+        CHECK_STATUS(mlssSetParameterByEnum(&context, opName, MLSS_ATTR_MHA_SCALE, &scale));
+        CHECK_STATUS(mlssSetParameterByEnum(&context, opName, MLSS_ATTR_MHA_DATATYPE, &data_type));
+        
+        MLSSstatus* pStatuses = NULL;
+        MLSSsize nStatuses = 0;
+
+        if (mlssGetCaps(context, &pStatuses, &nStatuses) != MLSS_SUCCESS)
+        {
+            std::cout << "Failed to get caps\n" << std::endl;
+        }
+        else
+        {
+            std::cout << "Got caps\n" << std::endl;
+            MLSSbinary* binaries = NULL;
+            MLSSsize n = 0;
+            CHECK_STATUS(mlssGetBinaries(context, &binaries, &n));
+            CHECK_STATUS(mlssPrintBinaries(binaries, n));
+            auto result = info.add_instruction(make_op("mlss_mha"), args);
+            return result;
+        }
```

Comment:
This is a backend operator and so it should not be added on the onnx frontend. We already fuse attention on the backend so you dont need to match a bunch of operators to see if its attention(but you will need to verify if it is not "extra" operators).